rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -51,53 +41,57 @@
 
 ipython_useSVG = False
 ipython_3d = False
```

Comment:
Would the end user change this to True to get 3D rendering and when is this done?

Would it be better to change this into private variable and use a setter and getter function? If the end user wants to use 3D rendering, we could print an error message if the notebook doesn't support it. The setter could set 
```
if _canUse3D:
    rdchem.Mol._repr_html_ = _toJSON
```
and the getter delete it.